common/api/endpoint/ApiEndPoint.ts

```diff
@@ -33,6 +33,8 @@ export const ApiEndPoint = {
 
     country: () => apiPath('country'),
 
+    review: () => apiPath('review'),
```

Comment:
add an object  Review and not under Assessment 
review : {
status : { many : ()=> apiPath('review','status')}
}